### ISSUE TITLE
[Feature Request] Data table translation for aria-label

### ISSUE THREAD

**Issue by TuntematonSotilas** [vuetifyjs/vuetify#5345]
### Problem to solve

Hello I encounter a problem with the data-table component
my issue is specific for visually handicapped persons
 

Here is my current Header generated by the component : 
```html
<th role="columnheader" scope="col" aria-label="Créée le: Sorted descending. Activate to remove sorting."</th>
```
In fact I would like to have the sort generated text : (ex : "Sorted descending. Activate to remove sorting") in the current langage of my application (french) 

### Proposed solution
Provide a way to custom aria-labels in the header object

**Comment by KaelWD** [comment]
There's a TODO for this: https://github.com/vuetifyjs/vuetify/blob/5eb4514eec8f45f6d5ef3dc51d5fae5db439e401/src/components/VDataTable/mixins/head.js#L126-L134

**Comment by nekosaur** [comment]
Fixed in 2.0
